spec/registrant_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+require './lib/registrant'
+
+
+RSpec.describe Registrant do
+    before(:each) do
+        @registrant_1 = Registrant.new('Bruce', 18, true)
+    end
+
+    it 'exists' do
+        expect(@registrant_1).to be_instance_of(Registrant)
+    end
+    describe 'attributes' do
+        it 'has attributes' do
+            expect(@registrant_1.name).to eq("Bruce")
+            expect(@registrant_1.age).to eq(18)
+            expect(@registrant_1.permit).to eq(true)
+            expect(@registrant_1.license_data).to eq({:license=>false, :renewed=>false, :written=>false})
+        end
+    end
+    describe 'methods work' do
```

Comment:
I would call this describe `#earn_permit` or maybe `#earn_permit changes return value of #permit?`